src/SWP/Bundle/AnalyticsBundle/Resources/config/services.yml

```diff
@@ -0,0 +1,14 @@
+services:
+    swp.analytics.requestmetric_repository:
```

Comment:
in https://github.com/superdesk/web-publisher/pull/76 I created some logic responsible for automatically registering repositories as a services independently from the selected persistence backend, but this can be merged like that for now if the mentioned PR wont be merged before this one